feat: scale tests import (#967)
NVIDIA/KAI-Scheduler: docs/scale-tests/README.md

# KAI Scheduler Scale Tests

## Overview

Scale tests validate KAI scheduler performance and correctness at large cluster sizes (hundreds to thousands of nodes). These tests simulate realistic workloads to ensure the scheduler maintains acceptable performance and correctness under scale.

## What We Test

Scale tests verify:

- **Scheduling performance**: Time to schedule large numbers of pods across many nodes

> - **Topology-aware scheduling**: Correct placement of distributed jobs with topology constraints

Reviewer: ```suggestion
- **Topology-aware scheduling**: Time to allocate for distributed jobs with topology constraints
```
